cli/azd/pkg/config/manager_test.go

```diff
@@ -19,7 +17,7 @@ func Test_SaveAndLoadConfig(t *testing.T) {
 		},
 	)
 
-	configFilePath, _ := GetUserConfigFilePath()
+	configFilePath := filepath.Join(t.TempDir(), "config.json")
```

Comment:
Since I now use configuration to save information about the currently logged in user, these tests had the side effect of "logging out" the current user.

So now they work against temporary files.

I think this would have also meant that any `azd` developer on our team that ran these unit tests would remove any of their configuration data, which seemed bad.